### PR TITLE
ani-cli update 4.0

### PR DESCRIPTION
**Pull request by candrapersada**
Closes #XXXX

Relates to #XXXX

- [ ] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/ani-cli.json [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
-    "version": "3.4",
+    "version": "4.0",
     "description": "A CLI to browse and watch anime. This tool scrapes the site gogoanime.",
     "homepage": "https://github.com/pystardust/ani-cli",
     "license": "GPL-3.0-or-later",
-    "url": "https://github.com/pystardust/ani-cli/releases/download/v3.4/ani-cli",
+    "url": "https://github.com/pystardust/ani-cli/releases/download/v4.0/ani-cli",
     "hash": "8dcefc128f239c2bb792986930deac947a6d5eda1815004832d999d80b719e68",
-    "depends": "axel",
+    "depends": "aria2",
```
Comment (port19x) [inline bucket/ani-cli.json:8]:
```suggestion
    "depends": "aria2", "fzf"
```

**Review by port19x — CHANGES_REQUESTED** [review]
The hash likely needs to be updated as well

File: bucket/ani-cli.json [hunk 2]
```diff
@@ -1,11 +1,15 @@
 {
-    "version": "3.4",
+    "version": "4.0",
     "description": "A CLI to browse and watch anime. This tool scrapes the site gogoanime.",
     "homepage": "https://github.com/pystardust/ani-cli",
     "license": "GPL-3.0-or-later",
-    "url": "https://github.com/pystardust/ani-cli/releases/download/v3.4/ani-cli",
+    "url": "https://github.com/pystardust/ani-cli/releases/download/v4.0/ani-cli",
     "hash": "8dcefc128f239c2bb792986930deac947a6d5eda1815004832d999d80b719e68",
```
Comment (port19x) [inline bucket/ani-cli.json:7]:
```suggestion
    "hash": "a98b78a3515c06014b572b3c49c1d8bfff6944d15b72fd8157c14ea1501ee0ab",
```

**Review by port19x — CHANGES_REQUESTED** [review]

**Review by port19x — APPROVED** [review]

**Comment by port19x** [comment]
- [x] someone
- [x] please
- [x] merge
- [x] (tasks to boost visibility lol)

**Comment by sitiom** [comment]
Superseded by #10494. As a rule, `suggest` is recommended over `depends` for use cases where the dependency is installed outside Scoop.

**Pull request closed without merging**